app/models/like.rb

```diff
@@ -1,2 +1,12 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  def self.update_counter(post_id, likes_count)
```

Comment:
[OPTIONAL]
- You can use the same suggestions I mentioned in the previous comment.